src/main/java/frc/robot/commands/SimpleRollerControl.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Feeder;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Launcher;
+
+/**
+ * A class for controlling the rollers (intake, feeder, and launcher) in a simple way. Each respective subsystem can optionally be added via chaining:
+ * 
+ * <code>
+ * new SimpleRollerControl().intake(m_intake, 0.5).feeder(m_feeder, 0.3);
+ * </code
+ */
+public class SimpleRollerControl extends Command {
```

Comment:
I added this class to make it easier to add these controls in RobotContainer. 

In the future, this class should hopefully not be needed after we have our more complex commands that will also control height and tilt automatically.